Controller/MailgunProjectSettings.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace Kanboard\Plugin\Mailgun\Controller;
+use Kanboard\Controller\BaseController;
+
+/**
+ *
+ * @author   Max Eisel
+ */
+class MailgunProjectSettingsController extends BaseController
+{
+    /**
+     * Mailgun Project Action Settings
+     *
+     * @access public
+     * @param array $values
+     * @param array $errors
+     */
+    public function show(array $values = array(), array $errors = array())
+    {
+        $this->response->html($this->helper->layout->project('MailgunProject:settings', array(
+            'owners' => $this->projectUserRoleModel->getAssignableUsersList($project['id'], true),
+	    'values' => array(
+                'MailgunProject_catchall'   => $this->projectMetadataModel->get($project['id'], 'MailgunProject_catchall'),
+                'project_id' => $_REQUEST['project_id'],
+		),
+            'project' => $project,
+            'title' => t('Edit Mailgun project settings')
+        )));
+    }
+
+    public function save()
+    {
+

```

Comment:
Useless blank line. Make sure to follow existing code style.

Reply:
Fixed - I didn't even realize that I added anything to that function